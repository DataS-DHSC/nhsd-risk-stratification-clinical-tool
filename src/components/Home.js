const Home = () => (
  <div className="nhsuk-grid-row">
    <div className="nhsuk-grid-column-two-thirds">
      <h1 id="qcovid-prediction-service-heading">
        QCovid&reg; Risk Prediction Service
      </h1>

      <p>Use this service to:</p>

      <ul>
        <li>
          Ensure patients and public know and understand their risk status
        </li>
        <li>Provide clinical support to the most vulnerable individuals*</li>
      </ul>
    </div>
  </div>
);

export default Home;
