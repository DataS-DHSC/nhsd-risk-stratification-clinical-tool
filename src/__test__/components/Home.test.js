import React from 'react';
import { render } from '@testing-library/react';
import Home from '../../components/Home';

const setup = () => render(<Home />);

describe('Home', () => {
  it('should match snapshot ', () => {
    const container = setup();
    expect(container.asFragment()).toMatchSnapshot();
  });
});
