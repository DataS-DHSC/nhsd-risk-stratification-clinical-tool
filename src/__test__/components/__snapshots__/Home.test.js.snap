// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match snapshot  1`] = `
<DocumentFragment>
  <div
    class="nhsuk-grid-row"
  >
    <div
      class="nhsuk-grid-column-two-thirds"
    >
      <h1
        id="qcovid-prediction-service-heading"
      >
        QCovidÂ® Risk Prediction Service
      </h1>
      <p>
        Use this service to:
      </p>
      <ul>
        <li>
          Ensure patients and public know and understand their risk status
        </li>
        <li>
          Provide clinical support to the most vulnerable individuals*
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
